_posts/2025-01-22-Introducing-OpenSearch-JS-Client-3.0.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title:  "Introducing OpenSearch JavaScript client 3.0"
+authors:
+ - theotr
+ - kolchfa
+date: 2025-01-09
+categories:
+ - release
+meta_keywords: javascript, typescript, client, api, opensearch, release, codegen, api generator
+meta_description: Release announcement for OpenSearch JS Client 3.0.0, a major overhaul of the client's API for better readability, consistency, and a more complete and accurate interface to interact with OpenSearch clusters.
```

Comment:
Please update the meta keywords and description with the following: 

_meta_keywords:_ OpenSearch JavaScript client 3.0, node.js client, typescript support, api generator, OpenSearch API 

_meta_description:_ Upgrade your OpenSearch development with JavaScript Client 3.0, featuring enhanced TypeScript support and consistency, automated updates, and an intuitive API interface for faster, more reliable applications.